src/main/kotlin/com/sourcegraph/cody/history/util/DurationGroupFormatter.kt

```diff
@@ -0,0 +1,25 @@
+package com.sourcegraph.cody.history.util
+
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+
+object DurationGroupFormatter {
+
+  fun format(since: LocalDateTime, now: LocalDateTime = LocalDateTime.now()): String {
+    val days = ChronoUnit.DAYS.between(since, now).toInt()
+    val weeks = ChronoUnit.WEEKS.between(since, now).toInt()
+    val months = ChronoUnit.MONTHS.between(since, now).toInt()
+    val years = ChronoUnit.YEARS.between(since, now).toInt()
+    return when {
+      days == 0 -> "Today"
+      days == 1 -> "Yesterday"
+      days in 2..6 -> "$days days ago"
+      months == 1 -> "Last month"
```

Comment:
Can you put `months` in right order relative to other durations? :)

Reply:
I agree that it looks counterintuitive.

However we can't sort this because it causes issues at the boundaries between month and week (for example "4 weeks" take precedence over "last month", see: `DurationGroupFormatterTest`).

Partially fixed + added comment to reduce confusion.